test-data/unit/check-type-aliases.test

```diff
@@ -1105,6 +1105,10 @@ t1: T1  # E: Variable "__main__.T1" is not valid as a type \
 T3 = TypeAliasType("T3", -1)  # E: Invalid type: try using Literal[-1] instead?
 t3: T3
 reveal_type(t3)  # N: Revealed type is "Any"
+
+T4 = TypeAliasType("T4")  # E: Missing positional argument "value" in call to "TypeAliasType"
+T5 = TypeAliasType("T5", int, str)  # E: Too many positional arguments for "TypeAliasType" \
+                                    # E: Argument 3 to "TypeAliasType" has incompatible type "Type[str]"; expected "Tuple[Union[TypeVar?, ParamSpec?, TypeVarTuple?], ...]"
```

Comment:
@JelleZijlstra Can I somehow suppress the second error here in the tests? It's not very useful (and not really correct - this is talking about the `type_params` keyword argument rather than any extra positional arguments)

Reply:
That seems like an unrelated bug (or at least, suboptimal error reporting) in mypy, where it compares the type of a positional argument with a keyword-only parameter. Here's a simple reproducer: https://mypy-play.net/?mypy=latest&python=3.12&gist=4f3a3447569518709568819cf8a61c4f

However, this is unrelated to the issue this PR is fixing, so I'd just leave this as is. If the unrelated issue gets fixed, we can just update this test.